.editorconfig

```diff
@@ -0,0 +1,17 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Once https://github.com/microsoft/vscode-jupyter-powertoys/pull/6 is addressed, these changes will either be a noop or will vanish from this PR